security/src/main/scala/pl/touk/nussknacker/ui/security/api/AuthenticatedUser.scala

```diff
@@ -1,6 +1,17 @@
 package pl.touk.nussknacker.ui.security.api
 
-final case class AuthenticatedUser(id: String, username: String, roles: Set[String])
+sealed trait BaseAuthenticationUserInfo {
```

Comment:
do we need it now?

Reply:
True, it's not needed anymore. I removed it